examples/graphql-api/.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+
+/.cache
+/build
+/public/build
```

Comment:
```suggestion
/public/build
/app/generated/types.ts
```

Reply:
So I went with a `/app/generated/*` as the codegen supports multiple outputs, seemed like it would grow better